swagger.yml

```diff
@@ -859,19 +904,20 @@ definitions:
         "timeline":
           [
             {
-              "event": "CREATED",
+              "eventName": "CREATED",
+              "eventDescription": "Order has been created",
```

Comment:
`event` does not exist on the response, so this needs to be updated to `eventName` (response in docs already reflect this)

also adding `eventDescription` too since that will exist on all timeline events now